messaging.rst

```diff
@@ -332,7 +325,7 @@ For the simplest Alice - Bob example:
 - Alice sends the ``RevealSecret`` to Bob and at this point she must assume the transfer is complete.
 - Bob receives the secret and at this point has effectively secured the transfer of ``n`` tokens to his side.
 - Bob sends a ``RevealSecret`` message back to Alice to inform her that the secret is known and acts as a request for off-chain synchronization.
-- Finally Alice sends an ``Unlock`` message to Bob. This acts also as a synchronization message informing Bob that the lock will be removed from the merkle tree and that the transferred_amount and locksroot values are updated.
+- Finally Alice sends an ``Unlock`` message to Bob. This acts also as a synchronization message informing Bob that the lock will be removed from the pending lock list and that the transferred_amount and locksroot values are updated.
```

Comment:
Again here, I am uncertain if it's a list with only one lock or if a list of locks.

Reply:
I'll change this to "the list of pending locks".

Reply:
Thanks. That's way better :)